deploy/examples/import-external-cluster.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+# run me like: . deploy/examples/import-external-cluster.sh
```

Comment:
Usage should be in the docs, not sure why we need it here?
```suggestion
```

Reply:
yaa not needed, now we have the clear instructions